ruby-gem/spec/dummy/app/javascript/routes/models/update-event.jsx

```diff
@@ -0,0 +1,37 @@
+import {digg, digs} from "diggerize"
+import {EventUpdated} from "@kaspernj/api-maker"
+
+export default class ModelsUpdateEvent extends React.Component {
+  state = {}
+
+  componentDidMount() {
+    this.loadFinishedTask()
+  }
+
+  render() {
+    const {connected, finishedTask} = this.state
+
+    return (
+      <div className="routes-models-update-event">
+        {finishedTask &&
+          <div className="finished-task-container" data-connected={connected} data-cache-key={finishedTask.cacheKey()}>
+            <div id="finished-task-name">
+              {finishedTask.name()}
+            </div>
+            <EventUpdated
+              onConnected={() => this.setState({connected: true})}
```

Comment:
Just to catch any random re-render issues.
```suggestion
              onConnected={this.onEventUpdatedConnect}
```